### ISSUE TITLE
Function arguments from JS to Java are null

### ISSUE THREAD

**Issue by djMax** [eclipsesource/J2V8#56]
```
myJavaFunction(function foo() {});
```

myJavaFunction always receives null as the first argument. (You said you already have a fix coming, figured I'd put in an issue to fix it against)

**Comment by irbull** [comment]
Thanks for the bug report. Here is a test case that demonstrates the problem.
```
    @Test
    public void testCallbackWithFunctionInParameterList() {
        ICallback callback = mock(ICallback.class);
        v8.registerJavaMethod(callback, "voidMethodWithObjectParameter", "foo", new Class<?>[] { V8Object.class });

        v8.executeVoidScript("var bar = function() {}; foo(bar);");

        verify(callback).voidMethodWithObjectParameter(isNotNull(V8Function.class));
    }
```
I have a fix in my workspace, I'll push it shortly and publish it to maven central tonight. Thanks!

**Comment by irbull** [comment]
Fixed with 49d48d8c942d4fdb83d946a7943936607cf4a6c8 and pushed to maven central.
